### ISSUE TITLE
Dont forget the page scroll

### ISSUE THREAD

**Issue by soulman-is-good** [Amomentforever0/components#5]
I did, but remembered in time ;P
https://github.com/Amomentforever0/components/blob/7a6f99eb0a7c7cbe72488fcbd3b2eae2ca6b7b6f/Tooltip/app.js#L52

**Comment by Amomentforever0** [comment]
> I did, but remembered in time ;P
> https://github.com/Amomentforever0/components/blob/7a6f99eb0a7c7cbe72488fcbd3b2eae2ca6b7b6f/Tooltip/app.js#L52

yes, i forgot about it, i'll add checks for boundaries like this
`if (left < 0) left = 0`
